src/cmd/hotspots/assert.rs

```diff
@@ -1,35 +1,47 @@
 use crate::{
     cmd::*,
-    result::{bail, Result},
+    result::Result,
     staking,
-    traits::{TxnEnvelope, TxnSign},
+    traits::{TxnEnvelope, TxnFee, TxnSign, TxnStakingFee},
 };
 
 #[derive(Debug, StructOpt)]
 /// Assert a hotspot location on the blockchain. The original transaction is
 /// created by the hotspot miner and supplied here for owner signing. Use an
 /// onboarding key to get the transaction signed by the DeWi staking server.
 pub struct Cmd {
-    /// Base64 encoded transaction to sign. If no transaction if given stdin is
-    /// read for the transaction. Note that the stdin feature only works if the
-    /// wallet password is set in the HELIUM_WALLET_PASSWORD environment
-    /// variable
-    #[structopt(name = "TRANSACTION")]
-    txn: Option<Transaction>,
+    /// Address of hotspot to assert
+    #[structopt(long)]
+    gateway: PublicKey,
+
+    /// Lattitude of hotspots location to assert
+    #[structopt(long)]
+    lat: f64,
 
-    /// The onboarding key to use if the payer of the transaction fees
-    /// is the DeWi "staking" server.
+    /// Longitude of hotspots location to assert
     #[structopt(long)]
-    onboarding: Option<String>,
+    lon: f64,
+
+    //// The antenna gain for the asserted hotspot.
```

Comment:
Which unit for the gain? dBi and dBd are both common.

Reply:
Done.. and made clear in the internal types too